### ISSUE TITLE
'updateUnreadCount' doesnt seem to be updated in iOS

### ISSUE THREAD

**Issue by nadavhalfon** [Fiksuruoka-fi/capacitor-intercom#17]
**Describe the bug**
'updateUnreadCount' doesnt seem to be updated in iOS.
I also debugged the swift code, maybe I'm doing something wrong

**To Reproduce**
```
Intercom.addListener('updateUnreadCount', (data: { unreadCount: number }) => {
  this.unreadIntercomCount = data.unreadCount;
});

Intercom.setupUnreadConversationListener();
Intercom.getUnreadConversationCount().then(({ unreadCount }) => {
  this.unreadIntercomCount = unreadCount;
});
```

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by eljass** [comment]
Fixed in: https://github.com/Fiksuruoka-fi/capacitor-intercom/pull/14/commits/e45b3deadd953a9fa7f14840ce8ca1c8e0679625

**Comment by eljass** [comment]
Fixed in 4.3.0
